libs/expo/betterangels/src/lib/screens/NotesHmis/HmisProgramNoteView/HmisProgramNoteTitle.tsx

```diff
@@ -0,0 +1,35 @@
+import { TextBold, TextRegular } from '@monorepo/expo/shared/ui-components';
+import { format } from 'date-fns';
+import { View } from 'react-native';
+import { HmisNoteQuery } from './__generated__/HmisProgramNoteView.generated';
+
+export default function HmisProgramNoteTitle({
+  hmisNote,
+}: {
+  hmisNote: HmisNoteQuery['hmisNote'] | undefined;
+}) {
+  if (hmisNote?.__typename !== 'HmisNoteType') return null;
+
+  return (
+    <View>
+      {hmisNote?.title && (
+        <TextBold size="lg" mb="xs">
+          {hmisNote?.title}
+        </TextBold>
+      )}
+      {hmisNote.date && (
+        <TextRegular mb="sm" size="sm">
+          {format(new Date(hmisNote.date), 'MM/dd/yyyy')}
+          {' @ '}
+          {format(new Date(hmisNote?.date), 'hh:mm a')}
+        </TextRegular>
+      )}
+      {/* {hmisNote.category && (
+        <>
+          <TextBold size="sm">Category</TextBold>
+          <TextRegular size="sm">{hmisNote.category}</TextRegular>
+        </>
+      )} */}
```

Comment:
Is this commented out code related to a future ticket?  If so can we link the code to it?